.github/workflows/go.yml

```diff
@@ -37,6 +37,37 @@ jobs:
     - name: Build
       run: cd ${{ matrix.module }} && go build ./...
 
+    - name: Install gVisor
```

Comment:
This is a lightly modified script from gVisor's installation instructions: https://gvisor.dev/docs/user_guide/install/